llzk-macro/src/parse/pass_set.rs

```diff
@@ -0,0 +1,40 @@
+//! Type representing a set of passes. Based on melior's.
+
+use super::IdentifierList;
+use proc_macro2::Ident;
+use syn::{
+    bracketed,
+    parse::{Parse, ParseStream},
+    LitStr, Result, Token,
+};
+
+pub struct PassSet {
```

Comment:
Can we add comments to the public structures and functions?

Reply:
Why is this resolved?

Reply:
I added the comment. For some reason it's not showing up in the conversation view but in the updated files view is visible